hack/shellcheck.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
Can we make this `#!/usr/bin/env bash` and add some `set -euxo pipefail`.

Reply:
Why? This is POSIX-compliant script which executes just one command. I don't see much benefit in searching and loading whole non-posix bash interpreter.

I have added `set -eux` as this is POSIX compliant and we don't have any pipes, so `-o pipefail` isn't improving anything.

Reply:
`set -eux` sounds good to me. `pipefail` would be just a precaution for future pipe based additions. The reason I asked for it was because I was bitten by this hard ;-)